superdesk/io/commands/update_ingest.py

```diff
@@ -226,6 +227,10 @@ def update_provider(self, provider, rule_set=None, routing_scheme=None):
             if items:
                 update[LAST_ITEM_UPDATE] = utcnow()
 
+        if CRT_LAST_UPDATED in provider:
```

Comment:
seems more like a hack, wouldn't it be better to pass this update dict to `feeding_service.update` call and let it do any changes there if needed?